frontend/src/routes/Upload.tsx

```diff
@@ -265,6 +292,13 @@ const UploadErrorBox: React.FC<{ error: unknown }> = ({ error }) => {
             probablyOurFault: true,
             potentiallyInternetProblem: false,
         };
+    } else if (error instanceof DOMException) {
+        info = {
+            causes: new Set([t("upload.errors.upload-cancelled")]),
+            probablyOurFault: false,
+            potentiallyInternetProblem: false,
+        };
+        failedAction = "";
```

Comment:
This feels a bit like a "cheat" to me. Maybe handling cancellation as an error isn't the best idea? :thinking: See also my overall review comment.